recipes/mygeotab/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mygeotab" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "48e29238da3ce8c236475fcf144fafeda785b3d8c7660beb9ded88732e0e63dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - requests
+    - click
+    - pytz
+    - python-dateutil
+    - six
+
+test:
+  imports:
+    - mygeotab
+
+about:
+  home: https://github.com/Geotab/mygeotab-python
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'An unofficial Python client for the MyGeotab API'
+  description: An Apache2 Licensed, unofficial Python client for the MyGeotab SDK.
+  doc_url: http://mygeotab-python.readthedocs.io/en/latest/
+  dev_url: https://github.com/Geotab/mygeotab-python
+
+extra:
+  recipe-maintainers:
+    - parthea
+    - aaront
```

Comment:
@aaront -- Are you ok being added as a co-maintainer of the `mygeotab` recipe on conda-forge? You were added by @parthea.